crates/wdl-ast/src/v1/task.rs

```diff
@@ -246,6 +254,8 @@ pub const TASK_FIELD_META: &str = "meta";
 pub const TASK_FIELD_PARAMETER_META: &str = "parameter_meta";
 /// The name of the `ext` task variable field.
 pub const TASK_FIELD_EXT: &str = "ext";
+/// The name of the `max_retries` task variable field.
+pub const TASK_FIELD_MAX_RETRIES: &str = "max_retries";
```

Comment:
@peterhuene I guess `max_retries` was not added before? Looking [at the spec](https://github.com/openwdl/wdl/blob/wdl-1.2/SPEC.md#runtime-access-to-requirements-hints-and-metadata), it indeed appears that `max_retries` is missing from the description of the `task` type. I'll make an issue.